services/121-service/src/payments/reconciliation/intersolve-voucher-reconciliation/intersolve-voucher-reconciliation.service.ts

```diff
@@ -18,11 +18,45 @@ export class IntersolveVoucherReconciliationService {
     private readonly intersolveVoucherScopedRepository: ScopedRepository<IntersolveVoucherEntity>,
   ) {}
 
+  private async updateVouchersWithErrorHandling(
+    vouchers: IntersolveVoucherEntity[],
+    programId: number,
+    credentials: any,
+    errorMessagePrefix: string,
+    incrementBeforeTry: boolean,
```

Comment:
I think this whole parameter and the split in behaviour can be removed; this function is called from 2 places, only one of which cares about the `updatedCount`.